packages/now-cli/test/dev/integration.js

```diff
@@ -1560,6 +1565,21 @@ test(
   })
 );
 
+test(
+  '[vercel dev] redwoodjs',
+  testFixtureStdio(
+    'redwoodjs',
+    async testPath => {
+      await testPath(200, '/', /<div id="redwood-app">/m);
+      await testPath(200, '/about', /<div id="redwood-app">/m);
+      const reqBody = '{"query":"{redwood{version}}"}';
+      const resBody = '{"data":{"redwood":{"version":"0.14.0"}}}';
+      await testPath(200, '/api/graphql', resBody, {}, 'POST', reqBody);
+    },
+    { isExample: true }
+  )
+);
```

Comment:
I want to start testing examples because currently these are untested in PRs 😬 

This `isExample` option was a quick way to get us there.

Reply:
These tests were copied from `probes` here: https://github.com/vercel/vercel/blob/74bb8a21ed710ccdc1546d474c2952181b58647f/packages/redwood/test/fixtures/01-create-redwood-app/vercel.json#L4-L19